data/tutorials/gs_00_up_and_running.md

```diff
@@ -18,10 +18,10 @@ reading [OCaml on Windows](/docs/ocaml-on-windows) first.
 The following instructions work on Linux, BSD, and macOS. Plus, they also work on Cygwin and
 WSL.
 
-**Guidelines for following instructions on this page**
+**Guidelines for Following Instructions on this Page**
 
-A **shell** is a program you can run that will let you enter commands in a text window using only your keyboard. When this page asks you to enter commands in your shell:
-On macOS, you will run the `Terminal` app to start a shell
+A **shell** is a program that will let you enter commands in a text windows using only your keyboard. It's also known as a command line interface (CLI). When this page asks you to enter commands in your shell, use the following instructions for your system:
```

Comment:
```suggestion
A **shell** is a program that will let you enter commands in a text window using only your keyboard. It's also known as a command line interface (CLI). When this page asks you to enter commands in your shell, use the following instructions for your system:
```

It will be "a text window" or "text windows" (without the article "a")